tests/integration/learning/test_domains.py

```diff
@@ -42,6 +42,7 @@ def registry_2(domain_2, test_registry):
     return ContractRegistry(MockRegistrySource(domain=domain_2))
 
 
+@pytest.mark.skip("inconsistent behaviour on CI - see #3289")
```

Comment:
This is an important regression test, I do not think it's good to keep it skipped for long.  #3289